doc/sphinx/language/core/records.rst

```diff
@@ -78,36 +150,44 @@ in which case the correctness of :n:`@type__3` may rely on the instance :n:`@ter
    Note here that the fields ``Rat_bottom_cond`` depends on the field ``bottom``
    and ``Rat_irred_cond`` depends on both ``top`` and ``bottom``.
 
-Let us now see the work done by the :cmd:`Record` command. First the command
-generates a variant type definition with just one constructor:
-:n:`Variant @ident {* @binder } : @sort := @ident__0 {* @binder }`.
-
-To build an object of type :token:`ident`, provide the constructor
-:n:`@ident__0` with the appropriate number of terms filling the fields of the record.
-
 .. example::
 
-   Let us define the rational :math:`1/2`:
+   Let us define the rational :math:`1/2` using either the applicative or record syntax:
 
     .. coqtop:: in
 
        Theorem one_two_irred : forall x y z:nat, x * y = 1 /\ x * z = 2 -> x = 1.
        Admitted.
 
        Definition half := mkRat true 1 2 (O_S 1) one_two_irred.
-       Check half.
 
-Alternatively, the following syntax allows creating objects by using named fields, as
-shown in this grammar. The fields do not have to be in any particular order, nor do they have
-to be all present if the missing ones can be inferred or prompted for
-(see :ref:`programs`).
+       Definition half' :=
+         {| sign := true;
+            Rat_bottom_cond := O_S 1;
+            Rat_irred_cond := one_two_irred |}.
```

Comment:
The definition seems more complex than necessary.  Perhaps it would be simpler to define `Rational` (rather than `Rat`) with only `top`, `bottom` and `sign` fields.

For `half'`, there should be some explanation about how `*_cond` binds `top` and `bottom`.  And maybe that should be a secondary example for maximum clarity.